pubspec.lock

```diff
@@ -415,4 +415,4 @@ packages:
     source: hosted
     version: "2.1.13"
 sdks:
-  dart: ">=2.0.0-dev <=2.0.0-dev.30.0"
+  dart: ">=2.0.0-dev <=2.0.0-edge.0d5cf900b021bf5c9fa593ffa12b15bcd1cc5fe0"
```

Comment:
I really don't know the consequences of committing this. I don't know how pub would know to compare two edge versions. If this is an issue, we should have a travis check here so that an edge constraint can't be committed.

Reply:
these cause zero issues. Just a bit weird to look at...